templates/apps/concentration-config.hbs

```diff
@@ -4,7 +4,7 @@
     <div class="form-group">
         <label>{{ localize "DND5E.AbilityModifier" }}</label>
         <select name="system.attributes.concentration.ability">
-            {{ selectOptions abilities selected=ability labelAttr="label" }}
+            {{ selectOptions abilities selected=ability blank="" labelAttr="label" }}
```

Comment:
Why? There's no affordance in the rules for not having an ability associated with a concentration save, is there?

Reply:
To allow to fall back to the default set in `CONFIG.DND5E.defaultAbilities` rather than having to adjust for each actor. This does not force the actor to make init/concentration without an ability.

Reply:
Ah ok, that makes sense